### PR TITLE
Add basic form layout

### PR DESCRIPTION
**Pull request by adrianammt**
Here is my first draft of the Settings Form. Design wise still work in progress. 
What I did:
- Create Form Component
- Display Form Component in the Create Page
- Save chosen settings to local storage

I look forward to your feedback! Also any ideas on how to make the form inputs more "App Like" are welcome.

### CODE REVIEW

**Review by namirsab — CHANGES_REQUESTED** [review]
Hey @adrianammt 
Cool UI and everything works with the local storage.
I've just made a few suggestions for the UI and for the code

One suggestion: make the label and the placeholder more different:
image

File: src/components/Main.css [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
   display: flex;
-  flex-wrap: wrap;
   flex-direction: column;
-  justify-content: space-around;
-  padding: 1em;
-  text-align: center;
-  background-color: violet;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: rgb(86, 59, 221);
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/components/Main.css:6]: Yes, it was on my todo list. I will.

File: src/components/SettingsForm.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  color: white;
+}
+
+.form-component {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid white;
+  height: 3em;
+}
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/components/SettingsForm.css:15]: Thank you for the tip!

File: src/components/SettingsForm.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+  useEffect(() => {
+    localStorage.setItem("classList", JSON.stringify(newClass));
+  }, [newClass]);
+
+  return (
+    <>
+      <form className="form" onSubmit={handleSubmit}>
+        <div className="form-component">
+          <label className="input-label">Name</label>
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/components/SettingsForm.js:50]: Totally oversaw that one. Thanks!

File: src/components/SettingsForm.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+          <input
+            type="text"
+            name="name"
+            id="name"
+            className="input-field name-field"
+            placeholder="Morning Meditation"
+            required
+          />
+        </div>
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/components/SettingsForm.js:59]: I will make a user story for this and put it in my backlog for next steps.

File: src/components/SettingsForm.js [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+        <div className="form-component">
+          <label className="input-label">Duration</label>
+          <input
+            type="time"
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/components/SettingsForm.js:63]: I actually need the seconds.

**Pull request merged**